tools/proofers/spellcheck_text.inc

```diff
@@ -125,54 +132,26 @@ function spellcheck_text($orig_text, $projectid, $imagefile, $aux_language, $acc
 
             $wordLen = UTF8::strlen($word);
 
-            // see if we are punctuation
-            if (in_array($word, $puncArray)) {
-                $newLine = UTF8::substr_replace($newLine, _wrapPunc($word), $lineIndex, $wordLen);
-            } elseif (in_array($word, $accepted_words)) {
-                // see if we're an AW word
-                $newLine = UTF8::substr_replace($newLine, _wrapAW($word), $lineIndex, $wordLen);
-            } else {
-                // not punctuation, handle word
```

Comment:
This code was lifted into the `_wrapBadWord()` function below and largely unchanged.